exploration/expression-attributes.md

```diff
@@ -204,38 +217,124 @@ the reserved/private-use rules will need to be adjusted to support attributes.
 
 ## Proposed Design
 
-Add support for option-like `@key=value` attribute pairs at the end of any expression.
+Solve the two disparate use cases separately,
```

Comment:
It might be helpful here to reiterate what the two disparate use cases are (attributes with a formatting runtime impact, and those without runtime impact).

Reply:
```suggestion
Provide separate solutions for attributes that impact formatting,
and those which do not,
```